### DISCUSSION TITLE
How to find App Data?

### DISCUSSION THREAD

**Discussion by LuckDragon666** [MaxiHuHe04/iTunes-Backup-Explorer#27] [General]
The fact I've got to use something like this to get my audio samples off the ipad is frustrating.

I've got some music apps that I'd like to get the samples off. I purchased the samples, and they are only usable in the app. The samples are royalty free, and paid for. I'm trying to do a batch download of all the files to use elsewhere, per the terms of use.

When I go into the Appdomain and look at the documents, a folder with 'packs' is listed, but no data is underneath it.

I'm trying to export my files from blocswav and remixlive.

I've exported the entire domain, and there are no music samples. 

What have I done wrong ? What do I need to do?

It's a non encrypted backup.

Edit : Turns out the software I'm using is not backing up the entire ipad. *sigh* Probably why the files aren't there. Will revisit this.

**Comment by MaxiHuHe04** [comment]
Hi @LuckDragon666,
which program did you use create the backup?
Unfortunately, i(Pad)OS backups are never really complete. Many system files are always missing and apps can mark files as excluded from backups, for example because they can be downloaded again anyway. Maybe they also do it intentionally to prevent what you tried.
If you already used an official way to backup the device, the files are probably excluded and can't be accessed this way.

Besides jailbreaking which might not be an option, if the app lets you remove and download the samples again, there is a chance you can use a man in the middle proxy. There are various ways the apps could have mitigated this and I won't go into exact details now, but that might be worth a try.
